stock_inventory_virtual_location/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Stock Inventory Virtual Location",
```

Comment:
Do you plan to add missing entries here (maintainers, dev.._status, ...) ?

Reply:
I don't know. I haven't thought about this yet.